internal/Util.hpp

```diff
@@ -214,6 +216,34 @@ static bool loadMoFileStringsToArray(FILE* moFile,
 	return true;
 }
 
+static std::vector<std::string> buildMoFilePaths(const char *domain)
+{
+	std::string languages = getenv("LANGUAGE");
+	std::vector<std::string> paths {""};
```

Comment:
There is an empty path in the result. Did you mean `std::vector<std::string> paths;` (i.e. an empty vector)?

Reply:
In my implementation, an empty string `""` would cause `loadMessageCatalog` try to open `moFilesRoot` as mo file first, perserves its original behavior.

Reply:
Thanks, I did not see that.